plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/Ck.hs

```diff
@@ -129,12 +134,17 @@ instance HasConstant (CkValue uni fun) where
 
     fromConstant = VCon
 
+-- CK not CEK
+-- substitution not environments
```

Comment:
Did you intend to leave these comments?

Reply:
You know, I have no idea where those are from